installation.md

```diff
@@ -17,10 +17,26 @@ After installing, `clangd --version` should print `clangd version 7.0.0` or late
 
 <details>
 <summary markdown="span">Mac OS X</summary>
+ 
 Clangd can be installed (along with LLVM) via [Homebrew](https://brew.sh):
 ```
 brew install llvm
 ```
+
+Alternatively, clangd is already bundled in Apple's [xcode command line tools](https://developer.apple.com/xcode/features/) if you installed it before via `xcode-select --install`. To check this, running `/Library/Developer/CommandLineTools/usr/bin/clangd --version` should print `clangd version 7.0.0` or later.
```

Comment:
Can I ask for an example of which version of command-line-tools you have installed, and what the corresponding `clangd --version` is? On the machine I have access to, the clang version string is `Apple LLVM version 10.0.1 (clang-1001.0.46.4)` which seems to be a fairly recent XCode 10.3 (July 2019), and doesn't come with clangd at all.

Reply:
```
~ ➜ xcode-select --version
xcode-select version 2373.
~ ➜ clang --version
Apple clang version 11.0.3 (clang-1103.0.32.59)
Target: x86_64-apple-darwin19.4.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
~ ➜ /Library/Developer/CommandLineTools/usr/bin/clangd --version
Apple clangd version 11.0.3 (clang-1103.0.32.59)
```
I'm on MacOS Catalina.

Reply:
I think most users should already have xcode-select, since it is a [requirement for homebrew installation](https://docs.brew.sh/Installation#macos-requirements), but I'm not sure about clangd being bundled in older versions...